src/main/java/com/treasure/hunt/strategy/hint/Product.java

```diff
@@ -1,4 +1,4 @@
-package com.treasure.hunt.strategy;
+package com.treasure.hunt.strategy.hint;
```

Comment:
Why is Product now exclusively for hints?

Reply:
I'm considering to add the hints into the game package as an interface in the next commit,
since for different game mods and/or add-ons, the seekers and tipsters need to implement custom instances of these.